installation/multi_node_debian.rst

```diff
@@ -59,15 +59,21 @@ Before starting the database let's change its access permissions. By default the
 
 .. note::
   Your DNS settings may differ. Also these settings are too permissive for some environments, see our notes about :ref:`worker_security`. The PostgreSQL manual `explains how <http://www.postgresql.org/docs/current/static/auth-pg-hba-conf.html>`_ to make them more restrictive.
+ Based on your DNS you can add another line with your IP address to allow the host unrestricted access, if you do not have an IP address that begins with 10.*.*.*. Otherwise add the line given for IPV4 address space. 
 
 **4. Start database servers, create Citus extension**
 
 ::
 
   # start the db server
   sudo service postgresql restart
+   or 
```

Comment:
Can you explain more about why we present alternatives here? When would someone choose to restart postgresql vs postgresql@14-main ?

Reply:
The idea is to ensure that someone does not start the server on another Postgres server. I had this issue where it started on a 9.5 server instead of 14. Someone would use postgres@14-main to ensure connection to postgres14, another person can use restart Postgres if they are sure their server is already on 14. Which would be the note about ps - ef. 
We could eliminate the option entirely and allow users to only use one.